apps/lost-stolen/acceptance/features/_happy-path.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+Feature('Lost or Stolen - Happy path');
+
+const values = {
+  'inside-uk': 'yes',
+  'use-address': 'true',
+  'org-help': 'yes'
```

Comment:
This works fine but I don't understand why there aren't more values.  I looked in the steps file and there are a bunch of fields but you've not put it here. e.g. in the personal details page date-of-birth, nationality etc

I looked at the documentation and it says 
```
completeToStep(step[, values]): Completes a form as far as a given step, using values specified or default values inferred from the field names.
```
Are there some default values that I can't see?